packages/sdk/src/transactions/OrdTransaction.ts

```diff
@@ -229,26 +226,29 @@ export class OrdTransaction {
       redeem: redeemScript
     })
 
-    // inscription tx always have 1 input and 1 + n custom outs
-    const fees = calculateTxFee({
-      totalInputs: 1,
-      totalOutputs: 1 + this.#outs.length,
-      satsPerByte: this.feeRate,
-      type: "taproot", // hardcoding because this process is only supported by Taproot txs
-      additional: { witnessScripts: inscribePayTx.witness }
+    this.#suitableUnspent = getDummyP2TRInput()
+    this.build()
+    const feeEstimator = new FeeEstimator({
+      psbt: this.psbt!,
+      feeRate: this.feeRate,
+      network: this.network
     })
+    const fee = feeEstimator.calculateNetworkFee()
```

Comment:
can we not remove the previous calculation of TxFee function?  can we create another function that still uses previous fee calculation? 
This is because every time we migrate our fee calculation, there is always minting that is stuck due to their commit using previous fee calculation and when revealing, the new fee fn is used causing different fees expected.